README.md

```diff
@@ -1,6 +1,6 @@
 # Consul Helm Chart
 
-## ⚠️	 ANNOUNCEMENT: Consul Helm Chart moving to `hashicorp/consul-k8s` ⚠️	
+## ANNOUNCEMENT: Consul Helm Chart moving to `hashicorp/consul-k8s`
```

Comment:
Maybe also make the 'ANNOUNCEMENT' in lower case? Could reduce horizontal size of the line.

Reply:
Yea I think the removal of the constructions alone won't turn this into a one liner.